addons/web/static/src/core/notifications/notification.variables.scss

```diff
@@ -0,0 +1,3 @@
+$o-notification-border-left-width: 0.75rem !default;
+$o-floating-box-shadow: 0 12px 14px -10px rgba(0, 0, 0, .25);
```

Comment:
`$o-floating-box-shadow` should be in primary_variables.scss 'cause it's not specific to any component.
`$o-dropdown-box-shadow` should remain in primary_variables.scss until we revamp the dropdown component... for now you can just update its value to `$o-floating-box-shadow`.